package.json

```diff
@@ -63,33 +63,33 @@
     "maplibre-gl": "^2.1.0"
   },
   "devDependencies": {
-    "@sveltejs/vite-plugin-svelte": "^1.0.0-next.49",
+    "@sveltejs/vite-plugin-svelte": "^1.0.1",
     "@tailwindcss/forms": "^0.5.2",
     "@tsconfig/svelte": "^3.0.0",
-    "@types/geojson": "^7946.0.8",
+    "@types/geojson": "^7946.0.10",
     "@types/mapbox__point-geometry": "^0.1.2",
     "@types/mapbox__polyline": "^1.0.2",
     "@types/mapbox__vector-tile": "^1.3.0",
-    "@typescript-eslint/eslint-plugin": "^5.30.3",
-    "@typescript-eslint/parser": "^5.30.3",
-    "autoprefixer": "^10.4.7",
-    "eslint": "^8.19.0",
+    "@typescript-eslint/eslint-plugin": "^5.32.0",
+    "@typescript-eslint/parser": "^5.32.0",
+    "autoprefixer": "^10.4.8",
+    "eslint": "^8.21.0",
     "eslint-plugin-svelte3": "^4.0.0",
     "husky": "^8.0.1",
     "lint-staged": "^13.0.3",
     "maplibre-gl": "^2.1.9",
-    "postcss": "^8.4.12",
+    "postcss": "^8.4.14",
     "postcss-load-config": "^4.0.1",
     "prettier": "^2.7.1",
     "prettier-plugin-svelte": "^2.7.0",
-    "rollup-plugin-visualizer": "^5.6.0",
+    "rollup-plugin-visualizer": "^5.7.1",
     "svelte": "^3.49.0",
     "svelte-check": "^2.8.0",
-    "svelte-preprocess": "^4.10.5",
-    "tailwindcss": "^3.1.4",
-    "tslib": "^2.3.1",
-    "typedoc": "^0.23.4",
+    "svelte-preprocess": "^4.10.7",
+    "tailwindcss": "^3.1.8",
+    "tslib": "^2.4.0",
+    "typedoc": "^0.22.18",
```

Comment:
Too many backwards-incompatible updates between 0.22 and 0.23. Rollback to use 0.22 is an easier fix.